config.yaml

```diff
@@ -102,11 +102,19 @@ Standards:
         Path: /02-react-week-1/react-tic-tac-toe.checkpoint.md
   - 
     Title: React Week 2
-    Description: Nested Components & Forms
-    UID: 1b8921be-9d97-4af0-ae04-2bda1cecc08b
+    Description: Hooks, Forms and API's
+    UID: 5b5f75d5-100b-4e75-849e-a8a1d8e4f425
     SuccessCriteria:
       - success criteria
     ContentFiles:
+      - 
+        Type: Lesson
+        UID: 5ae06a42-94f2-4927-8831-12987d7538fb
+        Path: /03-react-week-2/apis-get.md
+      -
+        Type: Lesson
+        UID: 0e9e569d-bd26-4c57-8b2d-77ccb15fcde9
+        Path: /03-react-week-2/apis-post.md
```

Comment:
should these should go after advanced forms based on the checklist Jared laid out?

Reply:
Yes, definitely. Thank you for catching that! I would love for us to be careful about ordering the lessons in the order we plan for students to consume them.

Reply:
I reordered it.